packages/application-generic/src/tracing/otel-wrapper.ts

```diff
@@ -0,0 +1,134 @@
+import { Span as OtelSpan } from 'nestjs-otel';
+import { TraceService as setTraceService } from 'nestjs-otel';
+import { MetricService as setMetricService } from 'nestjs-otel';
+import { OtelInstanceCounter as setOtelInstanceCounter } from 'nestjs-otel';
+import { OtelUpDownCounter as setOtelUpDownCounter } from 'nestjs-otel';
+import { OtelHistogram as setOtelHistogram } from 'nestjs-otel';
+import { OtelObservableGauge as setOtelObservableGauge } from 'nestjs-otel';
+import { OtelObservableCounter as setOtelObservableCounter } from 'nestjs-otel';
+import { OtelObservableUpDownCounter as setOtelObservableUpDownCounter } from 'nestjs-otel';
+import { OtelCounter as setOtelCounter } from 'nestjs-otel';
+import { MetricOptions, SpanOptions } from '@opentelemetry/api';
+import { Injectable } from '@nestjs/common';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function Span(name?: string, options?: SpanOptions) {
+  return OtelSpan(name, options);
+}
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function OtelInstanceCounter(options?: MetricOptions) {
+  return setOtelInstanceCounter(options);
+}
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function OtelUpDownCounter(
+  ...dataOrPipes: (
```

Comment:
We can extract this to an exported typing in this file, then reuse it throughout, e.g.

import { PipeTransform, Type } from '@nestjs/common';
export type OtelDataOrPipe = string | PipeTransform<any, any> | Type<PipeTransform<any, any>>;

...
dataOrPipes: OtelDataOrPipe[],

Reply:
Unless I am missing something that necessitates a dynamic import?

Reply:
I think you are right, I've pushed the changes